DQM/SiStripMonitorClient/python/SiStripSourceConfigTier0_cff.py

```diff
@@ -71,6 +66,11 @@
     errorReplyDcs = cms.bool( True ),
 )
 
+from Configuration.StandardSequences.Eras import eras
+if eras.stage2L1Trigger.isChosen():
+    SiStripMonitorClusterBPTX.BPTXfilter.stage2 = cms.bool(True)
+    SiStripMonitorClusterBPTX.BPTXfilter.l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis")
+    SiStripMonitorClusterBPTX.BPTXfilter.l1tExtBlkInputTag = cms.InputTag("gtStage2Digis")
```

Comment:
eras.xxx.isChosen syntax should not be used.
In this case it looks like eras.xxx.toModify syntax fits well
```python
eras.stage2L1Trigger.toModify(SiStripMonitorClusterBPTX, 
    BPTXfilter.stage2 = cms.bool(True)), 
    BPTXfilter.l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis"),
    BPTXfilter.l1tExtBlkInputTag = cms.InputTag("gtStage2Digis")
```